src/main.rs

```diff
@@ -70,6 +75,14 @@ fn run_target_program(elf_path: &Path, chip_name: &str, opts: &cli::Opts) -> any
     let elf = &Elf::parse(&elf_bytes, elf_path, reset_fn_address)?;
     let target_info = TargetInfo::new(elf, memory_map, probe_target, stack_start)?;
 
+    if let Some(table) = &elf.defmt_table {
+        if logger_info.has_timestamp() && !table.has_timestamp() {
+            log::warn!(
+                "logger format contains timestamp but no timestamp implementation was provided"
```

Comment:
Please add something like "Consider removing the timestamp (`{t}`) from your logger format or add a timestamp implementation (`defmt::timestamp!`)".

Reply:
Done